### ISSUE TITLE
Mentions of people should be hyperlinked

### ISSUE THREAD

**Issue by karthikraman** [RBC-DSAI-IITM/rbc-dsai-iitm.github.io#1]
e.g. Electives page: Clicking on "B. Ravindran" should go to the people page for B. Ravindran.

**Comment by abhisheknaik96** [comment]
The _ilds_ website had separate pages for faculty, students, collaborators. Should we have individual pages here for everyone associated, or follow the same format as before?

Secondly, we don't have the tags of people's interests as of now. Do we create the main 'People' page with just the photos (that we have) and their designations (no tags)?

**Comment by DenzilFrancisCrasta** [comment]
Currently information about people is serialized in Jekyll's **Data Files** which, to the best of my knowledge,  has no option to generate a page for each member in the people data file. 

Jekyll's **Collections** , however, offers serialization and individual pages for each member of the collection. 

We can use collections instead of the current data files to achieve what we desire. We will have individual pages for each member and we can hyperlink from any other page like electives using the permalinks. 

I can do this but will need some time as I have seminar ppt deadline at midnight.

**Comment by karthikraman** [comment]
This is not an urgent requirement, but should happen over time. Check the IBSE webpage, where this happens (at least in some places). For example: https://ibse-iitm.github.io/team/Karthik-Raman shows my papers, and the [news page](https://ibse-iitm.github.io/news/) shows the author names hyperlinked to the people page...

**Comment by karthikraman** [comment]
@abhisheknaik96, yes we need separate pages for everything possible. Again, see the organisation of https://github.com/IBSE-IITM/ibse-iitm.github.io/tree/master/team/_posts for example. It's so much easier to blame/track edits that way.

**Comment by DenzilFrancisCrasta** [comment]
I have pushed changes using jekyll's collections. 
We will need to add a file for each member under the _members folder.
Add any extra attributes like courses to the front matter of the individual files.
Css and image linking is currently broken which I will fix in some time

**Comment by abhisheknaik96** [comment]
I've scraped and added a lot of information about the people - faculty, collaborators. I couldn't figure out the flags for jekyll to look into subfolders inside _members, which @DenzilFrancisCrasta will look into.

**Comment by karthikraman** [comment]
Nice work.. Small note: have all `yml` files with `.yml` extenstion, rather than `.html`? Thanks!

**Comment by abhisheknaik96** [comment]
Done. Also fixed the People page with a new member attribute - faculty, collaborator.
